{"impact": {"pr": null, "a": "COMPLETE", "ac": "LOW", "impact_score": null, "i": "COMPLETE", "cvss3_score": null, "c": "COMPLETE", "vector_string": "AV:N/AC:L/Au:N/C:N/I:N/A:P", "cvss2_severity": "MEDIUM", "cvss3_severity": null, "s": null, "ui": null, "av": "LOCAL", "cvss2_score": 5.0, "au": "SINGLE", "exploitability_score": null}, "license": "HPND", "name": "pillow", "cve_id": "CVE-2014-3589", "repository_summary": "The friendly PIL fork (Python Imaging Library) https://python-pillow.org", "ochrona_severity_score": 5.0, "repo_url": "https://github.com/python-pillow/Pillow", "latest_version": "6.2.1", "cwe_id": "CWE-20", "language": "python", "publish_date": "2014-08-25T14:55Z", "references": ["http://lists.opensuse.org/opensuse-updates/2015-04/msg00056.html", "http://secunia.com/advisories/59825", "http://www.debian.org/security/2014/dsa-3009", "https://github.com/python-pillow/Pillow/commit/205e056f8f9b06ed7b925cf8aa0874bc4aaf8a7d", "https://pypi.python.org/pypi/Pillow/2.3.2", "https://pypi.python.org/pypi/Pillow/2.5.2"], "owner": "alex_clark", "affected_versions": [{"operator": "<=", "version_value": "2.3.2"}, {"operator": "=", "version_value": "2.5.0"}, {"operator": "=", "version_value": "2.5.1"}], "description": "PIL/IcnsImagePlugin.py in Python Imaging Library (PIL) and Pillow before 2.3.2 and 2.5.x before 2.5.2 allows remote attackers to cause a denial of service via a crafted block size."}